src/js/components/ConfigurationView.js

```diff
@@ -0,0 +1,242 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import DCOSStore from '../stores/DCOSStore'
+import DescriptionList from './DescriptionList';
+import Service from '../structs/Service';
+import StringUtil from '../utils/StringUtil';
+
+const sectionClassName = 'container-fluid container-pod container-pod-super-short flush flush-bottom';
+
+function fetchVersion(service, versionID) {
+  if (service.getVersions().get(versionID) == null) {
+    DCOSStore.fetchServiceVersion(service.getId(), versionID);
+  }
+}
+
+class ConfigurationView extends mixin(StoreMixin) {
+  constructor() {
+    super(...arguments);
+
+    this.store_listeners = [{name: 'dcos', events: ['change']}];
+  }
+
+  componentWillMount() {
+    super.componentWillMount();
```

Comment:
Actually, I don't see why this one is needed. In any case, StoreMixin only defines `didMount` and `willUnmount` LF methods.